hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -27,8 +27,13 @@ public static HawkBuilder init(Context context) {
     if (context == null) {
       throw new NullPointerException("Context should not be null");
     }
-    hawkBuilder = new HawkBuilder(context);
-    return hawkBuilder;
+
+    internal = null;
```

Comment:
sorry maybe it was not clear. by default it's null, yes. my point is to reset it null if its called again. so for instance after it has been built, if I call init I should have a new instance. hopefully makes sense to you.